__tests__/fixtures/action/deprecated_input_action.yml

```diff
@@ -0,0 +1,17 @@
+name: "An Action"
+description: "Test of deprecated inputs"
+author: "Niek Palm"
+inputs:
+  inputA:
+    required: false
+    description: This is an input
+    deprecationMessage: This input has been deprecated
```

Comment:
ref to docs: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddeprecationmessage